### PR TITLE
feat(wallet): create company wallet page

### PR DESCRIPTION
**Pull request by manojava-gk**
## Description

Company wallet listing page along with api integration

## Why

new feature

## Issue

https://github.com/eclipse-tractusx/portal-frontend/issues/443

## Checklist

- [x] I have followed the [contributing guidelines](https://github.com/eclipse-tractusx/portal-assets/blob/main/developer/Technical%20Documentation/Dev%20Process/How%20to%20contribute.md#commit-and-pr-guidelines)
- [x] I have performed a self-review of my own code
- [x] I have successfully tested my changes locally

### CODE REVIEW

**Comment by manojava-gk** [comment]
FYI - Zero linter issues in newly created folders

File: src/types/Constants.ts [hunk 1]
```diff
@@ -213,6 +214,7 @@ export enum ROLES {
   SUBMITTED_APPLICATION = 'view_submitted_applications',
   REQUEST_SSICREDENTIAL = 'request_ssicredential',
   DECISION_SSICREDENTIAL = 'decision_ssicredential',
+  COMPANY_WALLET = 'CX Admin',
```
Comment (manojava-gk) [inline src/types/Constants.ts:217]: @jjeroch Please provide role info

**Review by jjeroch — CHANGES_REQUESTED** [review]
- Grouping update needed
  - Group 1: "Frameworks" shows all credentials with credential type "Framework.*" 
  - Group 2: "Company Role" shows all credentials with credential type "CompanyRole.*"
  - Group 3: "Others" shows all other credentials which are responded
- where the BPNL is currently displayed, there the Credential Type should be shown (big and bold)
- Where the current credential type is displayed, there the BPNL should show up (small and not bold)
- Please mention "Expiry:" in front of the expiry date

**Comment by manojava-gk** [comment]
> * Grouping update needed
>   
>   * Group 1: "Frameworks" shows all credentials with credential type "Framework.*"
>   * Group 2: "Company Role" shows all credentials with credential type "CompanyRole.*"
>   * Group 3: "Others" shows all other credentials which are responded
> * where the BPNL is currently displayed, there the Credential Type should be shown (big and bold)
> * Where the current credential type is displayed, there the BPNL should show up (small and not bold)
> * Please mention "Expiry:" in front of the expiry date

Addressed the comments. but waiting for appropriate data to test @jjeroch

**Review by oyo — APPROVED** [review]

File: src/assets/locales/de/main.json [hunk 2]
```diff
@@ -1675,6 +1676,21 @@
...
+      "activateDescription": "You have activated your Catena-X wallet.",
+      "inactive": "Not Connected",
+      "inactiveDescription": "You have not activated your Catena-X wallet.",
+      "subnavigation": {
+        "title": "Wallet operations :",
+        "button1": "Request new Use Case Credential",
+        "button2": "Request new Company Credential",
+        "button3": "Get more details of VCs (coming soon)"
+      },
+      "expiry": "Expiry: "
```
Comment (jjeroch) [inline src/assets/locales/de/main.json:1693]: German Translation? ;)

**Review by jjeroch — CHANGES_REQUESTED** [review]

File: src/assets/locales/de/main.json (hunk 2 above)
Comment (manojava-gk) [reply src/assets/locales/de/main.json]: Check now

**Pull request merged**